unit_tests/ut_h_params.amlg

```diff
@@ -257,13 +257,11 @@
         (seq
             (system "system" "rmdir /S /Q ut_20test_5fmodel")
             (system "system" "del ut_20test_5fmodel.*")
-			(system "system" "del \"ut test_modelVersion.txt\"")
         )
 
         ;else posix
 		(seq
```

Comment:
no need for this outer seq anymore since it's a single statement

Reply:
Removed